src/theme/dark.ts

```diff
@@ -307,7 +307,11 @@ const controlBar = {
   shadow: shadows.large,
   bgd: colors.greys.grey100,
   border: 'none',
-  opacity: 1
+  opacity: 1,
+  active: {
```

Comment:
Is it confusing that sometimes `active` styling is applied when an element is actually in its active state (being clicked) and sometimes it means the property is selected? Maybe not, but just a thought.

Reply:
Fair point. Are you saying you'd prefer "selected'?

Reply:
I like `selected`. Looks good otherwise!

Reply:
I like it too. Looking at the `theme` file you're absolutely correct; we use `active` for the CSS :active selector, so it is confusing to use it in this context. Updating.